spec/nanoc/filters/colorize_syntax/rouge_spec.rb

```diff
@@ -53,12 +58,12 @@ def foo
         let(:output) do
           <<~EOS
             before
-            <pre><code class="language-ruby"><table style="border-spacing: 0"><tbody><tr>
-            <td class="gutter gl" style="text-align: right"><pre class="lineno">1
-            2</pre></td>
-            <td class="code"><pre>  <span class="k">def</span> <span class="nf">foo</span>
-              <span class="k">end</span><span class="w">
-            </span></pre></td>
+            <pre><code class="language-ruby"><table class="rouge-table"><tbody><tr>
```

Comment:
For the record this is not valid HTML. And is one of the reasons why I departed from `:colorize_syntax` to have my own `:rouge` filter.